tests/llm/utils/mock_toolset.py

```diff
@@ -56,6 +66,9 @@ def _get_mock_file_path(self):
 
     def _auto_generate_mock_file(self, params: Dict):
         mock_file_path = self._get_mock_file_path()
+
+        mock_file_path = deconflict_file_name(mock_file_path)
```

Comment:
Can we make the logic from make it eval reliable? or we will just override it when we will merge it?